Task2/Task2/Task2/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Task2.Products;
+using Task2.Services;
+using Newtonsoft;
+using Newtonsoft.Json;
+using Task2.Infrastructure;
+
+namespace Task2
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            var brick1 = new Brick("Brick Red", 3, 1.23, 5);
+            var brick2 = new Brick("Brick RED", 5, 1.25, 6);
+            
+            var brickResult1 = ServicesByProduct.AdditionOfTwoProducts(brick1, brick2);
+            var brickResult2 = brick1 + brick2;
+
+            List<Brick> listBricks = new List<Brick>
+            {
+                brick1,
+                brick2,
+                brickResult1,
+                brickResult2
+            };
+
+            foreach (var item in listBricks)
```

Comment:
Usually we don't use `item`, we just write more specific words like `brick`.

Reply:
fixed